app/assets/stylesheets/show.scss

```diff
@@ -16,3 +16,42 @@
     padding-right: 5px;
   }
 }
+
+.voting-title-container {
+  padding-top: 20px;
+  vertical-align: middle;
+}
+
+.voting {
+  float: left;
+  padding-right: 5px;
+  font-size: 25px;
+}
+
+.review-container {
+  float: left;
+}
+
+.review-title {
+  margin-top: 5px;
+}
+
+// .big {
+//   font-size: 50px;
+//   color: red;
+// }
+//
+// .small {
+//   font-size: 20px;
+// }
+
+.voted a{
+  color: red;
+  &:hover{
+  color: red;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 4 spaces, but was indented 2 spaces